docs/customer-360/customer-journey/process.sql

```diff
@@ -0,0 +1,24 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create stream of pages
+CREATE STREAM pages (
+  customer INTEGER,
+  time BIGINT,
+  page_id STRING,
```

Comment:
Just wondering: do we prefer `STRING` or `VARCHAR` (and are all recipes consistent -- or maybe it does not matter)?

(Personally I am very confused that we even support `STRING` as synonym to `VARCHAR` which seems to be the "proper" SQL type name?)

Reply:
@mjsax will track separately in https://github.com/confluentinc/ksqldb-recipes/issues/146